src/jit/optimizer.cpp

```diff
@@ -3063,19 +3070,25 @@ void Compiler::optUnrollLoops()
                 }
             }
 
+#ifdef JIT32_GCENCODER
+            if (fgReturnCount + loopRetCount > 4)
```

Comment:
It would be nice if we could convert this "4" as well as the one in loop cloning (and anywhere else we can find it) to something named, like MAX_GCENCODER_EPILOGS, either defined under JIT32_GCENCODER, or differently (possibly?) for all platforms. Note that loop cloner still restricts to 4 even for x64.